data/geolocation-cn

```diff
@@ -88,6 +88,7 @@ moji.com
 3322.org
 35.com
 360doc.com
+360jq.com
```

Comment:
why not add it to `qihoo360`

Reply:
http://www.360jq.com/about.htm

According to the business license, this domain does not belong to Qihoo 360

Reply:
> http://www.360jq.com/about.htm
> 
> According to the business license, this domain does not belong to Qihoo 360

sorry, my mistake